packages/functional-tests/tests/prod.smoke.spec.ts

```diff
@@ -9,10 +9,6 @@ test.describe('severity-1', () => {
     credentials,
     pages: { login, settings },
   }) => {
-    test.fixme(
-      true,
-      '(Invalid parameter in request body) response to attachedClientDisconnect mutation'
-    );
```

Comment:
and now that it's working we should finish off this test by checking that the row was removed from the ui

Reply:
A check that the row for the signed out device disappears is now in place. 

One minor thing to note here. After the initial login in this test, there are two devices. One row is for playwright and one row is for the browser being used to test. If the browser's device was logged out, then the browser would be directed back to the sign page. Because of this behavior, I now log out the playwright device instead of the browser. Doing so allows us to observe that the row for the signed out device disappears.

Reply:
oh, right! yes, this test is specifically meant for sync so we should assert that you're logged out